docs/en/ingest-management/faq.asciidoc

```diff
@@ -227,3 +226,22 @@ and port for your setup. If you run everything locally, the address is
 you can copy the {es} endpoint URL from the overview page of your deployment.
 If you're not running in {ecloud}, make sure the {kib} and {es} HTTPS ports
 are both accessible; by default these are `5601` and `9200` respectively.
+
+[discrete]
+[[how-do-i-reset-a-missing-dashboard-asset]]
+== If I delete a {kib} dashboard asset, how do I get it back?
+
+The aseets for a given integraion can be re-loaded into {es} with the API.
+You'll need to know the name of the package and the version you want to reinstall.
+Use cURL or your favorite API UI tool to run the following with the noted 'force' param
+in the body:
+
+POST /api/fleet/epm/packages/[package nmae]-[package version]
+{ "force": true }
+
+So, for example, to reset the system v1.0.0 package you could POST:
```

Comment:
I did like using the other words in case someone was searching for them - and Jen had used the term 'reset' in slack so I wanted it to be here.  I'm ok with whatever you feel is best.

Reply:
I see what you're saying. Inconsistencies add ambiguity, though, because users wonder, "Do all these words mean the same thing?"

Let's try to use the verb that is the most descriptive of what the user is trying to do. Hopefully that is the same term that users are likely to search on and the same term we use throughout the docs!